### PR TITLE
What's up with search?

### PR DESCRIPTION
**Pull request by barrucadu**
We'll use this, in addition to dropping messages into `#govuk-2ndline`, to update on the state of the migration.

---

[Trello card](https://trello.com/c/8B0HvaDY/100-prepare-some-documentation-while-the-migration-is-happening)

### CODE REVIEW

File: source/manual/whats-up-with-search.html.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+We (`#govuk-search-perf`) are upgrading GOV.UK's elasticsearch from version 2 to 5.  We're now rolling the change out to production.
+
+## What you need to know
+
+1. Alerts about search-api in AWS production are now important
+
+1. Please continue making changes to rummager, and then merging rummager `master` into search-api.  [Like this](https://github.com/alphagov/search-api/pull/28).
+
+1. If you want to update a content item *published* by rummager (eg, some finders, including the business readiness finder), you'll need to publish the ner version from rummager.  We'll be changing the `publishing_app` field of relevant content items later.
```
Comment (brucebolt) [inline source/manual/whats-up-with-search.html.md]:
```suggestion
1. If you want to update a content item *published* by rummager (eg, some finders, including the business readiness finder), you'll need to publish the new version from rummager.  We'll be changing the `publishing_app` field of relevant content items later.
```

**Comment by barrucadu** [comment]
Given that the new page is solely of use to 2ndline, it could all be on a card.  I don't really mind either way.

**Comment by brucebolt** [comment]
My thought is that a card is probably easier to update (since we don't have to get reviews on it)

**Comment by barrucadu** [comment]
Ok, I'll remove the new page and just leave a note about deploying the business readiness finder

**Pull request merged**